src/qt/rpcconsole.cpp

```diff
@@ -1118,6 +1118,7 @@ void RPCConsole::updateNodeDetail(const CNodeCombinedStats *stats)
     ui->peerDirection->setText(stats->nodeStats.fInbound ? tr("Inbound") : tr("Outbound"));
     ui->peerHeight->setText(QString("%1").arg(QString::number(stats->nodeStats.nStartingHeight)));
     ui->peerWhitelisted->setText(stats->nodeStats.m_legacyWhitelisted ? tr("Yes") : tr("No"));
+    ui->peerMappedAS->setText(stats->nodeStats.m_mapped_as == 0 ? tr("N/A") : QString("%1").arg(QString::number(stats->nodeStats.m_mapped_as)));
```

Comment:
```suggestion
    ui->peerMappedAS->setText(stats->nodeStats.m_mapped_as == 0 ? tr("N/A") : QString::number(stats->nodeStats.m_mapped_as));
```

Reply:
@promag tested, LGTM, done and extended to peerVersion and peerHeight